python/tests/README.md

```diff
@@ -0,0 +1,69 @@
+# Tests 
+This article discusses the tests for Konduit's Python SDK.
```

Comment:
@philip-khor since this is just a readme file so you don't have to write it as an article. Other than that just point out what's inside this readme. Something like, "Following is a quick description of what the test files contain" would better suit here.